### DISCUSSION TITLE
Reset single values instead of entire theme

### DISCUSSION THREAD

**Discussion by HitArrowLegend** [wintercms/winter#108] [Ideas]
Title explains it better than me. 

> If a user change the color of an element and want to go back to the default, it would be easier for them to just click a button to revert this one change rather than resetting everything

**Comment by bennothommo** [comment]
Hmmm, not sure how this would look in a UI perspective - it might add a bit of complexity to the form having a reset button or icon next to each field to reset it to the default. Thoughts?

**Reply by HitArrowLegend** [reply]
Well, to not clog up the backend, maybe a reset per page would be better ?
